database/sql/V071__drop_settlement_call_data.sql

```diff
@@ -0,0 +1,2 @@
+--- We don't need to store then uninternalised and internalised calldata, so this table can be dropped
```

Comment:
Let's explain a bit why it's no longer needed in the comment itself. Otherwise it's not very helpful.
```suggestion
--- We don't need to store the uninternalised and internalised calldata, so this table can be dropped
```

Reply:
Still no explanation why it's now fine to drop the table.